linters/README.md

```diff
@@ -5,3 +5,5 @@ Files to configure the linters we use for each language and technology
 
 * Ruby
   * [Rubocop](/linters/ruby/.rubocop.yml)
+* JavaScript
```

Comment:
Right, so it makes even more sense to have it explicit that it is for ES6, because it does not work on ES5, not ES7. No?